src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3078,6 +3079,12 @@ def update_vmss(cmd, resource_group_name, name, license_type=None, no_wait=False
         ScaleInPolicy = cmd.get_models('ScaleInPolicy')
         vmss.scale_in_policy = ScaleInPolicy(rules=scale_in_policy)
 
+    if spot_restore_enabled is not None:
+        vmss.spot_restore_policy.enabled = spot_restore_enabled
+
+    if spot_restore_timeout is not None:
+        vmss.spot_restore_policy.restore_timeout = spot_restore_timeout
```

Comment:
If the VMSS has not set spot restore policy before, can the null pointer error of `vmss.spot_restore_policy` appear here?

Reply:
If the VMSS has not set spot resotre policy before, it will have a default value from service response and the null pointer error will not appear here.

Reply:
OK, got it